include/tcpdf_static.php

```diff
@@ -1136,7 +1136,7 @@ public static function extractCSSproperties($cssdata) {
 	 * @see setHtmlVSpace()
 	 * @public static
 	 */
-	public static function fixHTMLCode($html, $default_css='', $tagvs='', $tidy_options='', &$tagvspaces) {
+	public static function fixHTMLCode($html, $default_css, $tagvs, $tidy_options, &$tagvspaces) {
```

Comment:
Is this a breaking change?

Reply:
Hi, its not. The last Parameter was never optional, so all user code must have defined those parameters. So all code use this function have already those params.